service/matching/version_rule_test.go

```diff
@@ -266,8 +266,7 @@ func TestInsertAssignmentRuleMaxRules(t *testing.T) {
 
 	// insert fourth --> error
 	_, err = insertAssignmentRule(mkAssignmentRule("1", nil), data, clock, 0, maxRules)
-	assert.Error(t, err)
-	assert.Equal(t, errExceedsMaxAssignmentRules(4, maxRules), err)
+	assert.ErrorContains(t, err, errExceedsMaxAssignmentRules(4, maxRules).Error())
```

Comment:
nit: this doesn't check that the error is the same type but that's probably fine.

Reply:
yeahh it's not my fave but I'm going to leave it, since ErrorIs didnt work